tests/test-verify-volume-mount.yml

```diff
@@ -1,9 +1,27 @@
 ---
+
+#
+# MDRaid devices paths are returned as a symlinks. But we need their targets.
+#
```

Comment:
Since this block appears in several places it should instead go into tests/verify-role-results.yml in the block titled 'Collect some information about the current state of the system."

You can also get the real/system device node by doing this:

```
- command: realpath "{{ storage_test_volume._device }}"
  register: storage_test_sys_node
```

(I did exactly this in the test result validation for swap in #90.)

Reply:
Partially fixed: Streamlined the code to use `realpath` instead. This part of code may be affected by upcoming mdraid volume PR which sometimes requires keeping symlink paths. I would rather do proposed change in that PR.